src/io/viint.h

```diff
@@ -0,0 +1,71 @@
+#ifndef _VIINT_H
+#define _VIINT_H
+#include "PR/os_internal.h"
+
+#define OS_TV_TYPE_PAL 0
+#define OS_TV_TYPE_NTSC 1
+#define OS_TV_TYPE_MPAL 2
+
+//TODO: figure out what this is
+#define VI_STATE_01 0x01
+#define VI_STATE_XSCALE_UPDATED 0x02
+#define VI_STATE_YSCALE_UPDATED 0x04
+#define VI_STATE_08 0x08         //related to control regs changing
+#define VI_STATE_10 0x10         //swap buffer
+#define VI_STATE_BLACK 0x20      //probably related to a black screen
+#define VI_STATE_REPEATLINE 0x40 //repeat line?
+#define VI_STATE_FADE 0x80       //fade
```

Comment:
```suggestion
#define VI_STATE_MODE_UPDATED 0x01
#define VI_STATE_XSCALE_UPDATED 0x02
#define VI_STATE_YSCALE_UPDATED 0x04
#define VI_STATE_CTRL_UPDATED 0x08   // related to control regs changing
#define VI_STATE_BUFFER_UPDATED 0x10 // swap buffer
#define VI_STATE_BLACK 0x20          // probably related to a black screen
#define VI_STATE_REPEATLINE 0x40     // repeat line?
#define VI_STATE_FADE 0x80           // fade
```
Some suggestions for the unnamed states, optional